ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -224,48 +214,47 @@ objects:
             NOW="$(date +%s)"
           done
           [[ "${NOW}" -ge "${TARGET}" ]] && echo "timeout waiting for ${ROUTER_NAMESPACE}/${ROUTER_DEPLOYMENT} to be available" && exit 1
+
+          # wait until the image registry changes propogate to the apiserver to avoid
+          # unnecessary restarts
+          until oc get is -n openshift php 2>/dev/null; do
```

Comment:
I [still][1] don't understand what this covers that the following query does not.

[1]: https://github.com/openshift/release/pull/2269#discussion_r237997396

Reply:
The image registry changes are separate than the kubeapiserveroperatorconfig/oauthconfig changes  (for configuring the kubeadmin user).  The oauth kubeadmin patch requires time to propagate to kube-apiserver.

Reply:
No, I mean the two `php` queries, [here][1] and [here][2].

[1]: https://github.com/openshift/release/pull/2410/commits/b6211490ffa92a1edc4d00922b1dc22ac94d9c10#diff-1c02c7c476c95d66377474c542fdfb33R220
[2]: https://github.com/openshift/release/pull/2410/commits/b6211490ffa92a1edc4d00922b1dc22ac94d9c10#diff-1c02c7c476c95d66377474c542fdfb33R223

Reply:
oh, gosh, I'm not sure! It's in the other cluster-launch-installer-* templates.  thinking....

Reply:
:woman_shrugging:  
I don't see any time when `oc get is -n openshift php` succeeds while this wouldn't `oc get is -n openshift php --template '{{ .status.dockerImageRepository }}'`?